src/celengine/dsooctree.cpp

```diff
@@ -43,7 +43,7 @@ double dsoAbsoluteMagnitudeDecayFunction(const double excludingFactor)
 
 
 template <>
-DynamicDSOOctree* DynamicDSOOctree::getChild(DeepSkyObject* const & _obj, const PointType& cellCenterPos)
+DynamicDSOOctree* DynamicDSOOctree::getChild(DeepSkyObject* & _obj, const PointType& cellCenterPos)
```

Comment:
reference to pointer?

Reply:
Well, the only thing I did was to remove `const`. For all the rest, ask original author, please :-)

Reply:
if you change something, cleanup please

Reply:
General implementation is done for references, while specialisation for DSOs uses pointers. It wouldn't be cleanup, but rewrite.